File: CodingPractice-master/leetcode/algorithms/python/1119_RemoveVowelsfrom_a_String.py
# -*- coding: utf-8 -*-
# @Author: Yan_Daojiang
# @Date:   2019-09-22 15:14:42
# @Last Modified by:   Yan_Daojiang
# @Last Modified time: 2019-09-22 15:14:51

class Solution:
    def removeVowels(self, S: str) -> str:
        vowels=['a','e','i','o','u']
        ans=""
        for i in range(0,len(S)):
            if S[i] not in vowels:
                ans=ans+S[i]
        return ans

File: CodingPractice-master/leetcode/algorithms/cpp/1119_RemoveVowelsFromAString.cc
class Solution {
public:
    string removeVowels(string S) {
        for (int i = 0; i < S.length(); ++i)
            if (S[i] =='a' || S[i] == 'e' || S[i] == 'i' || S[i] == 'o' || S[i] =='u'){
                --i;    //回退一步，防止连续元音
                for (int j = i + 1; j < S.length(); ++j)
                    S[j] = S[j+1];
            }
        return S;
    }
};